<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>every_z</class>
 <widget class="QWidget" name="every_z">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>677</width>
    <height>636</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="main_layout">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Coordinate selection (to be modified)</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="lyt_selector">
          <item>
           <widget class="QComboBox" name="combo_mappingCoor"/>
          </item>
          <item>
           <widget class="QPushButton" name="btn_start">
            <property name="text">
             <string>Start modification</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Coordinate modifications</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QGridLayout" name="lyt_modifications">
          <item row="5" column="0" colspan="3">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="btn_goToNext">
              <property name="text">
               <string>Next coordinate</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_goToPrev">
              <property name="text">
               <string>Previous coordinate</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="lbl_prevZ">
            <property name="text">
             <string>None</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>New Z-coordinate (µm):</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="lbl_coorLeft">
            <property name="text">
             <string>None</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0" colspan="3">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="btn_cancel">
              <property name="text">
               <string>Cancel modification</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_finishAndSave">
              <property name="text">
               <string>Finish and save modification</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Number of coordinates left to modify:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="3">
           <widget class="QCheckBox" name="chk_autoNextCoor">
            <property name="text">
             <string>Automove: Automatically go to the next coordinate after pressing &quot;Insert the current Z-coor&quot;</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QPushButton" name="btn_storeZ">
            <property name="text">
             <string>Insert current Z-coor</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDoubleSpinBox" name="spin_newZUm">
            <property name="minimum">
             <double>-1000000.000000000000000</double>
            </property>
            <property name="maximum">
             <double>1000000.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Original Z-coordinate (µm):</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="4" column="0" colspan="3">
           <widget class="QGroupBox" name="groupBox_3">
            <property name="title">
             <string>New Z-coordinate autofill</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QRadioButton" name="rad_lastFilled">
                 <property name="text">
                  <string>Use the last filled coordinate</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="rad_originalZ">
                 <property name="text">
                  <string>Use the original Z-coordinate</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_showInstructions">
       <property name="text">
        <string>Show instructions</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
